lib/sisimai/bite/email/zoho.rb

```diff
@@ -0,0 +1,167 @@
+module Sisimai::Bite::Email
+  module Zoho
+    # Sisimai::Bite::Email::Zoho parses a bounce email which created by Zoho Mail.
+    # Methods in the module are called from only Sisimai::Message.
+    class << self
+      # Imported from p5-Sisimail/lib/Sisimai/Bite/Email/Zoho.pm
+      require 'sisimai/bite/email'
+
+      Re0 = {
+        :'from'     => %r/mailer-daemon[@]mail[.]zoho[.]com\z/,
+        :'subject'  => %r{\A(?:
+             Undelivered[ ]Mail[ ]Returned[ ]to[ ]Sender
+            |Mail[ ]Delivery[ ]Status[ ]Notification
+            )
+        }x,
+        :'x-mailer' => %r/\AZoho Mail\z/,
+      }
+      Re1 = {
+        :begin  => %r/\AThis message was created automatically by mail delivery/,
+        :rfc822 => %r/\AReceived:[ \t]*from mail[.]zoho[.]com/,
+        :endof  => %r/\A__END_OF_EMAIL_MESSAGE__\z/,
+      }
+      ReFailure = {
```

Comment:
__[rubocop]__
* Style/MutableConstant: Freeze mutable objects assigned to constants.

<!-- Comment by [SideCI](https://sideci.com) -->